app/services/filter_tagged_content.rb

```diff
@@ -0,0 +1,62 @@
+class FilterTaggedContent
```

Comment:
I will rename if if that's clearer.

We have tried both ways of naming service objects (e.g. in content tagger: https://github.com/alphagov/content-tagger/tree/master/app/services/bulk_tagging) and I'm still not sure which one is more better, but happy to change it here.